bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -408,7 +419,6 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
             } else {
                 url = ajaxUrl + 'action_route=SaveFeature';
```

Comment:
Is this relying on a global variable? You should use Oskari.urls.getRoute('SaveFeature') instead.